src/Illuminate/Database/Schema/Grammars/Grammar.php

```diff
@@ -252,4 +253,21 @@ public function supportsSchemaTransactions()
     {
         return $this->transactions;
     }
+
+    /**
+     * Convert an array of column names into a delimited string.
+     *
+     * @param  array $columns
+     * @return string
+     */
+    public function columnizeWithKeys(array $columns)
```

Comment:
@tillkruss: I've created this separate method for building the expression of the columns in the index declaration considering `columnize()` is used in the query builder too and that the PR only concerned PostgreSQL modifiers at first. With that in mind, should I merge its code with a declaration of `columnize()` in this class (`\Illuminate\Database\Schema\Grammars\Grammar`)?